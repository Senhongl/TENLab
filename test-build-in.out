(1,.,.) = 
  0  0  0  0
  0  0  0  0
  0  0  0  0

(2,.,.) = 
  0  0  0  0
  0  0  0  0
  0  0  0  0
[ CPUIntType{2,3,4} ]
 3
[ CPUIntType{1} ]
 3
[ CPUIntType{1} ]
 2
 3
 4
 4
 5
 6
[ CPUIntType{6} ]
