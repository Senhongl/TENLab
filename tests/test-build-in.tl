a = [2,3,4];
b = zeros(a);
print(b);
c = [4,5,6];
print(shape(a));
print(cat(a,c,0));
