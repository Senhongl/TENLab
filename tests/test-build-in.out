(1,.,.) = 
  0  0  0  0
  0  0  0  0
  0  0  0  0

(2,.,.) = 
  0  0  0  0
  0  0  0  0
  0  0  0  0
[ CPUFloatType{2,3,4} ]